### DISCUSSION TITLE
I have problems installing packages

### DISCUSSION THREAD

**Discussion by manuel-marg** [termux/termux-app#3308] [Q&A] [answered]
I have a rather old phone running Android 5 and would love to practice a little Linux by installing Termux. Unfortunately current Termux versions are not compatible with my device as they require Android 9. But all is not lost, after much searching I found Termux version 0.79 works on Android 5 and opens perfectly. However, whenever I try to install any package, it gives me the error "Unable to locate package". I have been looking for solutions on various pages and on Youtube, and I always find the same solution: use the termux-change-repo command. But, in version 0.79, that command doesn't exist. I have tried to find a solution myself but so far no success. If anyone could help me fix this problem so I can install packages on my Termux, that would be a great help.

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6

**Reply by manuel-marg** [reply]
Hello Agnostic-Apollo! I just wanted to sincerely thank you for your help in solving my problem. Thanks to your answer, I was able to solve my problem successfully. I really appreciate your time and effort in helping me. I am very grateful to you!

**Reply by agnostic-apollo** [reply]
Welcome!
